t/cs.t

```diff
@@ -6,7 +6,7 @@ use Protocol::WebSocket::Frame;
 
 repeat_each(2);
 
-plan tests => repeat_each() * (blocks() * 4 + 3);
+plan tests => repeat_each() * (blocks() * 4 + 1);
```

Comment:
During each of the 3 commits, does commenting out `repeat_each()` above on L.9 still work given the updates to the tests plan on each commit?

Reply:
If we isolate these commits, propose them upstream eventually, these are the kind of things that would be blockers.

Reply:
Seems to work; is this what you meant?

```diff
-repeat_each(2);
-
-plan tests => repeat_each() * (blocks() * 4 + 1);
+plan tests => (blocks() * 4 + 1);
```

```
t/cs.t .. ok
All tests successful.
Files=1, Tests=149,  4 wallclock secs ( 0.03 usr  0.01 sys +  0.45 cusr  0.25 csys =  0.74 CPU)
Result: PASS
```

(frankly I'm surprised to not have messed it up because no matter how many times I read the docs, the test plan/count thing confuses the hell out of me)

Reply:
No I meant commenting out repeat_each

Reply:
That works as well (same output w/ 149 tests).

Reply:
Ok good, this needs to always be the case.